virttest/qemu_vm.py

```diff
@@ -1493,6 +1494,12 @@ def sort_key(dev):
                     cmdline = add_qxl(qxl_dev_nr, addr)
                     devices.insert(StrDev('qxl', cmdline=cmdline))
 
+        # Add watchdog device
+        if params.get("enable_watchdog", "no") == "yes":
+            device_type = params.get("watchdog_device_type", None)
```

Comment:
```suggestion
            device_type = params.get("watchdog_device_type")
```

Reply:
Make sense.  This is the old design, and i did not change the logic.  However, enable watchdog but without define device_type is not a correct usage, will update it.

Reply:
Make sense, updated.